pyvista/_vtk.py

```diff
@@ -0,0 +1,392 @@
+                                              vtkCylinderSource,
+                                              vtkSphereSource,
+                                              vtkPlaneSource,
+                                              vtkLineSource,
+                                              vtkCubeSource,
+                                              vtkConeSource,
+                                              vtkDiskSource,
+                                              vtkRegularPolygonSource,
+                                              vtkLineSource,
+                                              vtkPointSource,
+                                              vtkArrowSource)
+    from vtkmodules.vtkFiltersGeometry import (vtkGeometryFilter,
+                                               vtkStructuredGridGeometryFilter,
+                                               vtkCompositeDataGeometryFilter,
+                                               vtkDataSetSurfaceFilter)
+    from vtkmodules.vtkFiltersExtraction import (vtkExtractEdges,
+                                                 vtkExtractGeometry,
+                                                 vtkExtractGrid,
+                                                 vtkExtractSelection)
+    from vtkmodules.vtkFiltersTexture import (vtkTextureMapToPlane,
+                                              vtkTextureMapToSphere)
+    from vtkmodules.vtkFiltersPoints import (vtkGaussianKernel,
+                                             vtkPointInterpolator)
+    from vtkmodules.vtkFiltersVerdict import (vtkCellSizeFilter,
+                                              vtkCellQuality,
+                                              )
+    from vtkmodules.vtkImagingGeneral import vtkImageGaussianSmooth
+    from vtkmodules.vtkImagingCore import (vtkExtractVOI,
+                                           vtkImageExtractComponents,
+                                           vtkImageDifference,
+                                           vtkRTAnalyticSource)
+    from vtkmodules.vtkFiltersFlowPaths import vtkStreamTracer
+    from vtkmodules.vtkCommonExecutionModel import vtkImageToStructuredGrid
+
+    from vtkmodules.numpy_interface.dataset_adapter import (VTKObjectWrapper,
+                                                            numpyTovtkDataArray,
+                                                            VTKArray)
+    from vtkmodules.util.numpy_support import (vtk_to_numpy,
+                                               numpy_to_vtk,
+                                               numpy_to_vtkIdTypeArray,
+                                               get_vtk_array_type)
+
+    from vtkmodules.vtkCommonCore import (buffer_shared,
+                                          vtkAbstractArray,
+                                          vtkWeakReference)
+
+    # lazy import for some of the less used readers
+    # lazy import for some of the less used readers
```

Comment:
```suggestion
```
Missed this duplication on my first skim, sorry.

The style for the imports (empty lines, placement of closing parentheses)  might have to be unified before the PR is merged, but this is just bikeshedding. Heroic work you've done on this!

Reply:
Good catch.

> The style for the imports (empty lines, placement of closing parentheses) might have to be unified before the PR is merged

I'd like to keep the imports on this module one per line for readability since we're placing a ton.  As for empty lines, there might have been a method to the madness at some point; does it make sense to remove all empty lines?  We can do that and since I'm waiting for @banesullivan to review, we should clean this up.

Reply:
One on each line is good, I agree. There are two violations of this rule:
```sh
$ git grep -nE '^\s*from.+,[^$]' pyvista/_vtk.py 
pyvista/_vtk.py:44:    from vtkmodules.vtkIOPLY import vtkPLYReader, vtkPLYWriter
pyvista/_vtk.py:221:    from vtkmodules.vtkCommonMath import vtkMatrix4x4, vtkMatrix3x3
```

I think both "always empty lines" and "no empty lines" would work.

And similarly, I don't have a very strong opinion about closing parentheses, but now there are three styles being mixed:
```py
    from vtkmodules.vtkRenderingOpenGL2 import (vtkOpenGLHardwareSelector,
                                                vtkRenderStepsPass,
                                                vtkEDLShading,
                                                vtkOpenGLRenderer,
                                                vtkOpenGLTexture)
    [snip]
    from vtkmodules.vtkIOImage import (vtkBMPReader,
                                       vtkDEMReader,
                                       vtkDICOMImageReader,
                                       vtkDICOMImageReader,
                                       vtkJPEGReader,
                                       vtkJPEGReader,
                                       vtkMetaImageReader,
                                       vtkNrrdReader,
                                       vtkNrrdReader,
                                       vtkPNGReader,
                                       vtkPNMReader,
                                       vtkSLCReader,
                                       vtkTIFFReader,
                                       vtkTIFFReader,
    )
    [snip]
    from vtkmodules.vtkRenderingAnnotation import (vtkScalarBarActor,
                                                   vtkCornerAnnotation,
                                                   vtkAxesActor,
                                                   vtkAnnotatedCubeActor,
                                                   vtkLegendBoxActor,
                                                   vtkCubeAxesActor,
                                                   )

Reply:
If I were being pedantic I think this is the part of PEP 8 that is relevant (excerpt):
> The closing brace/bracket/parenthesis on multiline constructs may either line up under the first non-whitespace character of the last line of list, as in:
> 
> ```
> my_list = [
>     1, 2, 3,
>     4, 5, 6,
>     ]
> ```
> or it may be lined up under the first character of the line that starts the multiline construct, as in:
> ```
> my_list = [
>     1, 2, 3,
>     4, 5, 6,
> ]
> ```

But arguably imports would look weirder, being pulled in to the left like that. So I think the third option above is fine.